rageval/metrics/_answer_claim_recall.py

```diff
@@ -1,24 +1,18 @@
 # -*- coding: utf-8 -*-
 
-import pytest
-import numpy as np
-import pandas as pd
-from abc import ABC
 from typing import List, Any, Callable
 
-
 import datasets
-# from datasets import Dataset
+import numpy as np
 from dataclasses import dataclass
 
 from rageval.metrics import Metric, add_attribute
-from rageval.utils import text_to_sents
 
 
 _DESCRIPTION = """\
-AnswerNLIGroundedness is the metric build based on NLI model.
+AnswerClaimRecall is the metric build based on NLI model.
```

Comment:
Use the following description: """The AnswerClaimRecall is to measure the correctness of long-form answers. In the original paper, the author first use Instruct-GPT(text-davinci-003) to generate three "sub-claims" (based on gold answers) and use a state-of-the-art natural-language inference (NLI) model TRUE(Honovich et al., 2022) to check whether the model output entails the sub-claims (claim recall). """

Reply:
Maybe the metric name should be renamed as "AnswerNLICorrectness" to keep consistence with other AnswerCorrectness metrics, e.g., "AnswerRougeCorrectness" and "AnswerEMCorrectness".